pkg/logging/log.go

```diff
@@ -24,8 +24,11 @@ import (
 
 func Info(output string, args ...interface{}) {
 	blue := color.New(color.FgBlue).SprintFunc()
-	fmt.Print(blue("Info: "))
-	fmt.Printf(output, args...)
+	if output != "" {
+		fmt.Print(blue("Info: "))
+		fmt.Printf(output, args...)
+	}
```

Comment:
I think a blank line will still be helpful if it's blank
```suggestion
	} else {
		fmt.Println()
	}
```

Reply:
there is already new line https://github.com/rook/kubectl-rook-ceph/blob/df1ab8b44787e1286eddeb4b46938edd43b58fb3/pkg/logging/log.go#L26-L32

Reply:
oh right, I overlooked it, thanks